app/controllers/host_reports_controller.rb

```diff
@@ -2,6 +2,28 @@
 
 class HostReportsController < ::ApplicationController
   include Foreman::Controller::AutoCompleteSearch
+  def index
+    respond_to do |format|
+      format.html do
+        render '/react/index'
+      end
+
+      format.json do
```

Comment:
Since we have `index` action in non-api controller, I guess we can actually re-use it on Host reports index page instead of using API controller. It's not in the scope of this PR, just as a reminder for future refactoring.

Reply:
sounds good! I believe we could even use the same component